docs/development/end-to-end-testing.md

```diff
@@ -34,3 +34,29 @@ You need to make sure to install the official [cucumber](https://marketplace.vis
 
 Then you will have all the autocomplete features available to you
 
+### Running e2e tests locally
+
+Running e2e tests locally unfortunately requires some extra steps to make it work with our current setup. please don't commit any of these changes, create a stash of it instead so you can easily apply it when needed.
+
+The different docker containers need to be able to communicate with each other. This is not possible when you use localhost. You need to use the host.docker.internal hostname instead.
+
+`docker-compose.yml`
+- replace all the "localhost" with "host.docker.internal"
```

Comment:
Hmm, dat is strikt genomen niet nodig. Niet alle 'localhosts' hoef je te veranderen. Sterker nog, het ging er maar om eentje volgens mij, waar we vorige week achterkwamen? Ik weet niet meer welke.